guides/common/modules/proc_applying-configurations.adoc

```diff
@@ -0,0 +1,42 @@
+[id="Applying_configurations_{context}"]
+= Applying configurations
+
+In following sections we suggest various tunables and how to apply them.
+Please always test changing these in non production environment first, with valid backup and with proper outage window as in most of the cases Satellite restart is required.
+
+It is also a good practice to setup a monitoring before applying any change as it will allow you to evaluate effect of the change.
+Our testing environment might be too far from what you will see although we are trying hard to mimic real world environment.
+
+If you have changed some systemd service file, you need to notify systemd daemon to reload the configuration:
+
+----
+# systemctl daemon-reload
+----
+
+Note this will still need to reload the service to do the actual change. It is best to restart whole Satellite:
+
+[options="nowrap" subs="attributes"]
+----
+# {foreman-maintain} service restart
+----
+
+If you have changed some configuration file like `/etc/foreman-installer/custom-hiera.yaml` you need to rerun installer to apply the changes:
+
+[options="nowrap" subs="attributes"]
+----
+# {foreman-installer}
+----
+
+Also you might want to rerun installer with some new option, so you can:
+
+[options="nowrap" subs="attributes,quotes"]
+----
+# {foreman-installer} _new options_
+----
```

Comment:
```suggestion
```

To me, it is unclear _why_ this is part of the procedure.


Also, the whole procedure is missing `.Procedure` and a numbered list.

Reply:
Right, will reword it.

Regarding making it numbered list: It is not  a single procedure I would say. It is more set of steps that are connected, but based on what exactly you are changing, you need to pick right subset.

Reply:
Added, thank you!